spec/system/purchase_system_spec.rb

```diff
@@ -49,6 +49,15 @@
           expect(page).to have_text(dollar_value(purchases.sum(&:amount_spent_in_cents)))
           expect(page).to have_text(dollar_value(3579))
         end
+
+        it "User sees comments" do
```

Comment:
This doesn't need to be a system spec - it can be a request spec, which is more lightweight.

Reply:
Thanks for flagging that, should've done that from the get go 👍🏻